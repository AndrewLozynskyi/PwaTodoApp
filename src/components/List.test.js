import React from 'react';
import { shallow } from 'enzyme';
import List from './List';

describe('List', () => {
  it('renders without crashing', () => {
    const component = shallow(<List />);
    expect(component).toMatchSnapshot();
  });

});
