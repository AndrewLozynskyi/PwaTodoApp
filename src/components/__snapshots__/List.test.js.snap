// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List renders without crashing 1`] = `
<div
  className="to-do-list-container"
>
  <div
    className="row"
  >
    <div
      className="col s10 offset-s1 center-align"
    >
      <h4>
        You have no more things ToDo!
      </h4>
    </div>
  </div>
  <div
    className="fixed-action-btn"
  >
    <button
      className="btn-floating btn-large black darken-4"
      onClick={[Function]}
    >
      <i
        className="large material-icons"
      >
        +
      </i>
    </button>
  </div>
</div>
`;
